rpcs3/Emu/RSX/GL/GLTexture.cpp

```diff
@@ -319,6 +319,14 @@ namespace rsx
 				std::vector<gsl::byte> staging_buffer)
 			{
 				int mip_level = 0;
+				if (is_compressed_format(format))
+				{
+					//Compressed formats have a 4-byte alignment
+					//TODO: Verify that samplers are not affected by the padding
+					width = align(width, 4);
+					height = align(height, 4);
```

Comment:
The vertices are fine (when the cast is changed to u32).

On Vulkan the rendering of the 512x2 texture was ok before this change, the issue only affected OpenGL. So I guess this alignment fix broke Vulkan. I will look at it in Renderdoc and compare with my change.